vertx-web-graphql/pom.xml

```diff
@@ -32,8 +32,8 @@
     <!--
     Update these properties in the vertx-lang-* modules too
      -->
-    <graphql.java.major.version>14</graphql.java.major.version>
-    <graphql.java.version>${graphql.java.major.version}.0</graphql.java.version>
+    <graphql.java.major.version>16</graphql.java.major.version>
+    <graphql.java.version>${graphql.java.major.version}.1</graphql.java.version>
```

Comment:
As a personal preference (and I guess it would be easier for product builds), I'd just use a single property:

```
<graphql.version>14.1</graphql.version>
```